### DISCUSSION TITLE
Is difference merging to a new different model from a trained model the same as training the new model?

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1314] [Q&A] [answered]
will difference merging give inferior results than directly training?

**Comment by TheLastBen** [comment, marked as answer]
It really depends on the model and the trained subject, you'll have to do some testing to find what best suits your needs.

**Reply by gilroff** [reply]
ok thanks

**Reply by Omenizer** [reply]
I did difference merging once or twice and got lots of artifacts, so I prefer to train directly

**Reply by gilroff** [reply]
oh ok thanks
